### PR TITLE
JNG-3861 Using Spring PlatformTransactionManager

### PR DESCRIPTION
**Pull request by robertcsakany**
<table>
<td>
  <a href="https://blackbelt.atlassian.net/browse/JNG-3861" title="JNG-3861" target="_blank">StoryJNG-3861</a>  Transaction support for SDK with JSL DSL
  </td></table>
  <br />
 

JNG-3861 Using Spring PlatformTransactionManager

### CODE REVIEW

File: judo-runtime-core-dispatcher/pom.xml [hunk 1]
```diff
@@ -87,6 +88,13 @@
             <artifactId>validation-api</artifactId>
         </dependency>
 
+        <!-- https://mvnrepository.com/artifact/org.springframework/spring-tx -->
+        <dependency>
+            <groupId>org.springframework</groupId>
+            <artifactId>spring-tx</artifactId>
+            <version>4.3.12.RELEASE</version>
```
Comment (noherczeg) [inline judo-runtime-core-dispatcher/pom.xml:95]: this should come from a property?

File: pom.xml [hunk 2]
```diff
@@ -331,7 +331,7 @@
                     </execution>
                 </executions>
             </plugin>
-
+<!--
```
Comment (noherczeg) [inline pom.xml:334]: Shouldn't this be removed?

**Review by noherczeg — CHANGES_REQUESTED** [review]

File: pom.xml (hunk 2 above)
Comment (robertcsakany) [reply pom.xml:334]: No, it have to be fixed.

File: pom.xml [hunk 3]
```diff
@@ -338,7 +339,7 @@
                 <version>1.18.20.0</version>
                 <configuration>
                     <sourceDirectory>${project.basedir}/src/main/java</sourceDirectory>
-                    <outputDirectory>${project.basedir}/target/generated-sources/delombok</outputDirectory>
+                    <outputDirectory>${project.basedir}/target/delombok</outputDirectory>
```
Comment (bencegelei) [inline pom.xml:342]: nice catch

File: pom.xml [hunk 4]
```diff
@@ -359,7 +360,7 @@
                     <source>8</source>
                     <detectJavaApiLink>false</detectJavaApiLink>
                     <sourcepath>
-                        ${project.basedir}/target/generated-sources/delombok,
+                        ${project.basedir}/target/delombok,
```
Comment (bencegelei) [inline pom.xml:363]: nice catch

**Review by bencegelei — APPROVED** [review]

**Pull request merged**